test/functional/readconcern_tests.js

```diff
@@ -1,18 +1,11 @@
 'use strict';
 const setupDatabase = require('./shared').setupDatabase;
+const filterForCommands = require('./shared').filterForCommands;
 const expect = require('chai').expect;
 
-let client;
-let url;
-
-function filterForCommands(commands, bag) {
-  commands = Array.isArray(commands) ? commands : [commands];
-  return function(event) {
-    if (commands.indexOf(event.commandName) !== -1) bag.push(event);
-  };
-}
-
 describe('ReadConcern', function() {
+  let client;
```

Comment:
Why do we need this up here? Could we instead move `let client;` to inside of `'client-url specific ReadConcern'`?